src/Microsoft.OpenApi.Readers/V2/OpenApiDocumentDeserializer.cs

```diff
@@ -228,7 +229,18 @@ private static void FixRequestBodyReferences(OpenApiDocument doc)
                 var walker = new OpenApiWalker(fixer);
                 walker.Walk(doc);
             }
+        }
 
+        internal static IOpenApiExtension LoadExtension(string name, ParseNode node)
+        {
+            if (node.Context.ExtensionParsers.TryGetValue(name, out var parser))
+            {
+                return parser(node.CreateAny());
+            }
+            else
+            {
+                return node.CreateAny();
+            }
         }
```

Comment:
This should be in [https://github.com/Microsoft/OpenAPI.NET/blob/master/src/Microsoft.OpenApi.Readers/V2/OpenApiV2Deserializer.cs](https://github.com/Microsoft/OpenAPI.NET/blob/master/src/Microsoft.OpenApi.Readers/V2/OpenApiV2Deserializer.cs)